src/views/AfhandelingView.vue

```diff
@@ -1,13 +1,8 @@
 <template>
```

Comment:
There's a type import in this file that doesn't exist.

Reply:
fixed